pull/github.go

```diff
@@ -318,6 +347,49 @@ func (ghc *GitHubContext) loadPagedData() error {
 	return nil
 }
 
+func (ghc *GitHubContext) loadCollaboratorData() error {
+	// this is a minor optimization: make max(c,r) requests instead of c+r
+	var q struct {
+		Repository struct {
+			Collaborators struct {
+				PageInfo v4PageInfo
+				Edges    []struct {
+					Permission string  `graphql:"permission"`
+					Node       v4Actor `graphql:"node"`
+				}
+			} `graphql:"collaborators(first: 100, after: $collaboratorCursor, affiliation: DIRECT)"`
+		} `graphql:"repository(owner: $owner, name: $name)"`
+	}
+	qvars := map[string]interface{}{
+		"owner": githubv4.String(ghc.owner),
+		"name":  githubv4.String(ghc.repo),
+
+		"collaboratorCursor": (*githubv4.String)(nil),
+	}
+
+	collaborators := make(map[string]string)
+	for {
+		complete := 0
```

Comment:
You can remove all use of `complete` in this function. It is used in `loadPagedData` because there are multiple sequences, potentially with different lengths, that are paged together. Here there's only a single sequence.